src/status_im/contexts/profile/data_store.cljs

```diff
@@ -3,3 +3,7 @@
 (defn accepted-terms?
   [accounts]
   (some :hasAcceptedTerms accounts))
+
+(defn testnet?
```

Comment:
I see only one usage of this function and think it is better to be placed in events, near to its use.
`data-store` is a place where conversion from lower layers happens, I think the presence of `db` here is unnecessary, wdyt?

Reply:
update: probably my lsp glitches, now I see more usages of `profile/testnet?`. Still don't think data-store is a right place for it.

Reply:
@clauxx, we have `wallet/db.cljs` and this could be a good place to have functions that work with the db structure. Those functions can be reused between events and subs to avoid duplication. Probably we can have more `db.cljs` under different contexts. Wdyt?

Reply:
yeah, that would be a good place :+1: